src/zenml/exceptions.py

```diff
@@ -173,6 +172,8 @@ def __init__(
             parameters_class: Class of the parameters object for which
                 the parameters are missing.
         """
+        import textwrap
```

Comment:
only thinking about this since it came up earlier today when we were talking. I'm not entirely sure `textwrap` from the `stdlib` is the best way to do this? Or I'm not sure if there's not a better way using `rich` or our cli_utils etc?

Reply:
Can fully ignore this...

Reply:
I am open to any suggestions tbh. I didn't know any alternatives so I left it as it is.